README.md

```diff
@@ -11,6 +11,33 @@ Simply add "europeana-i18n" to your Rails app's Gemfile and `bundle`.
 Translations from the YAML locales in this gem will then be available to your
 app.
 
+## JavScript Translations
+
+Javascript translations are enabled through the [I18n-js](https://github.com/fnando/i18n-js) gem. 
+You will need to add these js files to your app/assets/javascript/application.js
+
+```
+//= require i18n_initializer
+//= require i18n
+//= require i18n_translations
+```
+
+Then ensure you load the translations specific to your current locale.
+This gem automatically generates translation files per local. (see config/i18n-js.yml)
+In order to know which language file to use include the JsTranslationsHelper and call the js_tranlation_files method.
+
+```
+class YourClass
+  include Europeana::I18n::JsTranslationsHelper
+  
+  def self.required_js_files
+    js_translation_files
+  end
+end
+
+YourClass.required_js_files #=> ["/javascripts/i18n/en.js"]
```

Comment:
This example would not work. `required_js_files` is a class method so would not have access to the instance method `js_translation_files`.